server/models/video/video-channel-sync.ts

```diff
@@ -0,0 +1,163 @@
+import { isVideoChannelSyncStateValid } from "@server/helpers/custom-validators/video-channel-syncs"
+import { CONSTRAINTS_FIELDS, VIDEO_CHANNEL_SYNC_STATE } from "@server/initializers/constants"
+import { VideoChannelSync, VideoChannelSyncState } from "@shared/models"
+import { MChannelSyncFormattable } from "@server/types/models/video/video-channel-sync"
+import { AttributesOnly } from "@shared/typescript-utils"
+import {
+  AllowNull,
+  BelongsTo,
+  Column,
+  CreatedAt,
+  DataType,
+  Default,
+  ForeignKey,
+  Is,
+  Model,
+  Table,
+  UpdatedAt
+} from "sequelize-typescript"
+import { AccountModel } from "../account/account"
+import { getSort, throwIfNotValid } from "../utils"
+import { VideoChannelModel } from "./video-channel"
+import { ActorModel } from "../actor/actor"
+import { Op } from "sequelize"
+import { UserModel } from "../user/user"
+import { isVideoImportTargetUrlValid } from "@server/helpers/custom-validators/video-imports"
+
+type AvailableForListOptions = {
+  accountId: number
+  search?: string
+  host?: string
+  handles?: string[]
+  forCount?: boolean
+}
+
+@Table({
+  tableName: 'videoChannelSync'
+})
+export class VideoChannelSyncModel extends Model<Partial<AttributesOnly<VideoChannelSyncModel>>> {
+
+  @AllowNull(false)
+  @Default(null)
+  @Is('VideoChannelExternalChannelUrl', value => throwIfNotValid(value, isVideoImportTargetUrlValid, 'externalChannelUrl', true))
+  @Column(DataType.STRING(CONSTRAINTS_FIELDS.VIDEO_CHANNEL_SYNCS.EXTERNAL_CHANNEL_URL.max))
+  externalChannelUrl: string
+
+  @CreatedAt
+  createdAt: Date
+
+  // We have no use of this field
```

Comment:
Please remove this comment. Even if we don't use this field, I think it can be interesting in the future.

Also, please add a `lastSyncAt` field that you update every time you check/run the sync